core/src/main/java/com/graphhopper/routing/ch/PrepareContractionHierarchies.java

```diff
@@ -46,6 +46,7 @@
  * @author Peter Karich
  */
 public class PrepareContractionHierarchies extends AbstractAlgoPreparation implements RoutingAlgorithmFactory {
+    public static final String STALL_ON_DEMAND = "stall_on_demand";
```

Comment:
Should this better go into the Parameters class?

Reply:
Or should we probably hide this config and do not even make it public? (At least it needs the `ch.` prefix)

Reply:
it was in Parameters.Algorithms before :) Maybe Parameters.CH would be a good place for it ?

Reply:
I think it is useful to be able to deactivate it for example in the Measurement class

Reply:
:D ... oh ... so yes, lets keep it here but reduce visibility? I currently do not see a case where it would make sense to disable stalling except for debugging and performance measurements. And then we could use the pure string without introducing some public usable constants.

Reply:
I usually prefer constants so I can search for usages, but you probably want to keep the public API as small as possible to reduce compatibility problems in future versions I guess. In this case yes I would make the constant private or remove it and just use the string value where it is needed.

Reply:
Yes, exactly. The only solution would be package protected but would not help much here.